bin/check-package-version

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+var exec = require( 'child_process' ).exec;
+var packageJsonPath = require( '../' + process.argv[2] );
+var semver = require( 'semver' );
+
+if ( packageJsonPath.private ) {
+	process.exitCode = 0;
+}
+
+exec( 'npm view ' + packageJsonPath.name + ' version', function( error, stdout ) {
```

Comment:
Resolved in:  https://github.com/Automattic/wp-calypso/commit/152aa0fafee09c8ce94497a9184aea0c4b3e5dad